pkgs/development/libraries/rocksdb/default.nix

```
@@ -26,11 +28,14 @@ stdenv.mkDerivation (finalAttrs: {
     hash = "sha256-Zifn5Gu/4h6TaEqSaWQ2mFdryeAarqbHWW3fKUGGFac=";
   };
 
+  patches = [ ./fix-findliburing.patch ];
```

```suggestion
  patches = [ ./fix-findliburing.patch ];
```
What is the source of this patch?
If the source is a URL, make it a `fetchpatch` and mention the source/PR with a comment in that case.